CHANGES.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## 1.11.0 (core) / 0.27.0 (libraries)
+
+## Major changes since 1.10.0 (core) / 0.26.0 (libraries)
+
+### Components — Configurable, reusable building blocks for data pipelines (now stable)
+
+Components, first released as Preview in 1.10.6, have reached Release Candidate status. APIs are stable, fully documented, and are the recommended, product-ready defaults for new Dagster projects.
+
+- **Declarative spec:** A short YAML block in `defs.yaml`, or a lightweight `Component` subclass in Python, lets you spin up assets, resources, schedules, IO managers, and checks, removing boilerplate while keeping every definition type-checked.
+- **Custom Components:** Subclassing `Component` lets you wrap any internal script or SaaS behind a strongly-typed interface; get the same autocompletion and docs as first-party integrations.
+- **Pythonic Templating:** Register variables or helpers with `@template_var` so pipeline authors edit parameters directly in YAML without touching Python. Optional inline components keep small bits of Python co-located.
+- **Powerful tooling:** High quality errors, strongly typed schema, tooling used in CI/CD, and auto-generated reference docs in UI.
```

Comment:
Maybe: "High quality errors, strongly-typed schemas, rich CLI support, and auto-generated docs directly in the UI"

What's the tooling used in CI/CD bit? Might stay away from that just given the overlap of "Powerfool tooling" / "tooling used in CI/CD"

Reply:
but yeah strongly-typed schemas == tooling used in CI/CD